core/src/main/java/org/bitcoinj/core/LockTime.java

```diff
@@ -119,7 +119,7 @@ public long rawValue() {
     }
 
     /**
-     * This is equivalent to {@code lockTime instanceof HeightLock || lockTime instanceof TimeLock}.
```

Comment:
I think this expression would now require proper brackets.

However, maybe better use human words:

> The lock time is considered to be set only if its raw value is greater than zero. In other words, it is set if it is either a non-zero block height or a timestamp.